src/main/java/com/example/springboot/controller/ShoppingListController.java

```diff
@@ -0,0 +1,37 @@
+package com.example.springboot.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.springboot.service.Product;
+import com.example.springboot.service.ShoppingListService;
+
+@RestController
+@RequestMapping("/api/v1/item")
+public class ShoppingListController {
+	@Autowired
+	private ShoppingListService lista = new ShoppingListService();
+	
+	
+	@DeleteMapping
+	public ResponseEntity<Integer> removeProduct(@RequestParam("name") String name,List<Product> ls) {
```

Comment:
The list cannot be provided by the user. The list must be inside the app, inside the service